Source/WebKit/UIProcess/WebProcessProxy.cpp

```diff
@@ -2208,6 +2209,36 @@ void WebProcessProxy::getNotifications(const URL& registrationURL, const String&
     WebNotificationManagerProxy::sharedServiceWorkerManager().getNotifications(registrationURL, tag, sessionID(), WTFMove(callback));
 }
 
+void WebProcessProxy::setAppBadge(std::optional<WebPageProxyIdentifier> identifier, const SecurityOriginData& origin, std::optional<uint64_t> badge)
```

Comment:
identifier should be named pageID as per the pattern in this file. identifier is too generic.